pysma/definitions.py

```diff
@@ -510,6 +511,70 @@
         daily_yield,
         pv_gen_meter,
     ],
+    DEVCLASS_INVERTER_TRI: [
+        status,
+        pv_power_a,
+        pv_power_b,
+        pv_voltage_a,
+        pv_voltage_b,
+        pv_current_a,
+        pv_current_b,
+        grid_power,
+        frequency,
+        current_l1,
+        current_l2,
+        current_l3,
+        voltage_l1,
+        voltage_l2,
+        voltage_l3,
+        power_l1,
+        power_l2,
+        power_l3,
+        total_yield,
+        daily_yield,
+        pv_gen_meter,
+        metering_power_absorbed,
+        metering_frequency,
+        metering_total_yield,
+        metering_total_absorbed,
+        metering_current_l1,
+        metering_current_l2,
+        metering_current_l3,
+        metering_voltage_l1,
+        metering_voltage_l2,
+        metering_voltage_l3,
+        metering_active_power_feed_l1,
+        metering_active_power_feed_l2,
+        metering_active_power_feed_l3,
+        metering_active_power_draw_l1,
+        metering_active_power_draw_l2,
+        metering_active_power_draw_l3,
+        battery_voltage_a,
+        battery_current_a,
+        inverter_power_limit,
+        battery_power_charge_total,
+        battery_power_discharge_total,
+        grid_reactive_power,
+        grid_reactive_power_l1,
+        grid_reactive_power_l2,
+        grid_reactive_power_l3,
+        grid_apparent_power,
+        grid_apparent_power_l1,
+        grid_apparent_power_l2,
+        grid_apparent_power_l3,
+        grid_power_factor,
+        battery_charge_total,
+        battery_discharge_total,
+        battery_soc_total,
+        battery_capacity_total,
+        battery_temp_a,
+        inverter_condition,
+        operating_status_general,
+        battery_status_operating_mode,
+        grid_relay_status,
+        inverter_system_init,
+        current_total,
+    ],
```

Comment:
The battery and metering sensors should not be here. They will be added dynamically.

Reply:
ok I will remove them
and test the code on my inverter